package/batocera/core/batocera-splash/S29abortsplash

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# Stop Splash Screen
+
+[[ $1 == start ]] || exit 0
+
+# We can set here an additional timer to extend videoplay
+# This value can be setted to batocera.conf
+splash_autoplay ()
+{
+    count=0
+    while [[ $count -lt 90 ]]
+    do
+        sleep 1
```

Comment:
S30 wouldn't run until video is finished. that's why i would like to have kill in S31emulation station

Reply:
What is S30? I don't have this service on RPi

Reply:
I think we dont use it atm. When possible, we use & to dispatch scripts as soon as possible . Even in S31 if i am not mistaken. So, boot would hang here and other scripts would not execute.

Reply:
Yes but it depends on the service that will be loaded after S29.
From my use case it tooks from S03 to S29 maybe 3-4 seconds only. The most workload is EmulationStation which leads to a black screen for round about 10 seconds maybe less maybe more.

So I think we should not pay attention to this. The users then can set the splash like they want. Some key values like to disable/enable the splash screens at all can be nested to batocera.conf --> they will be available to batocera-boot.conf after first reboot. So I think we can even do some finetuning and show a dedicated "PLEASE WAIT .... BATOCERA is INITIATING" boot screen.